tests/test_regression.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from cleanlab.regression import rank
+
+# To be used for all the tests
+labels = np.array([1, 2, 3, 4])
+predictions = np.array([1, 3, 4, 5])
+
+# test with deafault parameters
+def test_output_shape_type():
+    scores = rank.get_label_quality_scores(labels=labels, predictions=predictions)
+    assert labels.shape == scores.shape
+    assert isinstance(scores, np.ndarray)
+
+
+# test for acceptable datatypes
+@pytest.mark.parametrize("format", [pd.Series, pd.DataFrame, list])
+def test_type_error_for_input_types(format):
+    with pytest.raises(TypeError) as error:
+        _ = rank.get_label_quality_scores(labels=format(labels), predictions=format(predictions))
```

Comment:
What assertions can we make with the `error` in this `pytest.raises` context?

At a bare minimum, I'd typically just check if the expected error message is printed to make sure we raised the error at the appropriate point.

Reply:
@krmayankb always assert that the expected error message is being printed

Reply:
more generally though, why can we not make the code work for pd.Series or lists?  All you would need to do is call: `np.asarray(labels)` to convert it

Reply:
Added support for ArrayLike. Tests are also modified to take care of this. Please refer latest update.